### PR TITLE
ConcurrencyLimiter polishing and cleanup

### PR DESCRIPTION
**Pull request by DylanDmitri**
renamed: RequestThrottling -> ConcurrencyLimiter

Cleaned up sample project.

### CODE REVIEW

File: src/Middleware/ConcurrencyLimiter/sample/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        Random rnd = new Random();
+
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
+        {
+            app.UseConcurrencyLimiter();
+            app.Run(async context =>
+            {
+                var delay = rnd.Next(15, 85);
+                //var delay = 100;
```
Comment (halter73) [inline src/Middleware/ConcurrencyLimiter/sample/Startup.cs]: Nit: remove commented-out code

Comment (halter73) [reply src/Middleware/ConcurrencyLimiter/sample/Startup.cs]:
```suggestion
```

File: src/Middleware/RequestThrottling/RequestThrottling.slnf [hunk 2]
```diff
@@ -16,10 +16,11 @@
       "..\\http\\Headers\\src\\Microsoft.Net.Http.Headers.csproj",
```
Comment (halter73) [inline src/Middleware/RequestThrottling/RequestThrottling.slnf]: This should be deleted now, right?

Comment (halter73) [reply src/Middleware/RequestThrottling/RequestThrottling.slnf]: Make sure that the RequestThrottling folder is completely removed.

Comment (Tratcher) [reply src/Middleware/RequestThrottling/RequestThrottling.slnf]: Need to rename and move the slnf file.

**Comment by halter73** [comment]
It looks like you're still doing some renames. Ping me when it's ready for final review, but this looks good so far.

File: src/Middleware/RequestThrottling/RequestThrottling.slnf (hunk 2 above)
Comment (DylanDmitri) [reply src/Middleware/RequestThrottling/RequestThrottling.slnf]: yes it should

File: src/Middleware/ConcurrencyLimiter/sample/Startup.cs (hunk 1 above)
Comment (DylanDmitri) [reply src/Middleware/ConcurrencyLimiter/sample/Startup.cs]: I want to have this here for the demo

File: src/Middleware/ConcurrencyLimiter/sample/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddStackQueue((options) => {
+                options.MaxConcurrentRequests = Environment.ProcessorCount;
+                options.RequestQueueLimit = 50;
+            });
+        }
+
+        //Random rnd = new Random();
```
Comment (halter73) [inline src/Middleware/ConcurrencyLimiter/sample/Startup.cs]: Remove the commented out code

**Review by halter73 — APPROVED** [review]

File: src/Middleware/ConcurrencyLimiter/ConcurrencyLimiter.slnf [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "solution": {
-    "path": "..\\Middleware.sln",
+    "path": "C:\\Users\\t-dygra\\Desktop\\AspNetCore\\src\\Middleware\\Middleware.sln",
```
Comment (dougbu) [inline src/Middleware/ConcurrencyLimiter/ConcurrencyLimiter.slnf]: This should not have been checked in. Please do a follow-up PR to unbreak other users.

File: src/Middleware/ConcurrencyLimiter/src/QueuePolicies/QueuePolicyOptions.cs [hunk 5]
```diff
@@ -19,6 +19,6 @@ public class QueuePolicyOptions
         /// Maximum number of queued requests before the server starts rejecting connections with '503 Service Unavailible'.
         /// Defaults to 5000 queued requests.
```
Comment (dougbu) [inline src/Middleware/ConcurrencyLimiter/src/QueuePolicies/QueuePolicyOptions.cs]: Isn't the default now '0'?

**Pull request merged**